docker/deepsea/openSUSE-15.0/Dockerfile-base

```diff
@@ -1,7 +1,7 @@
 FROM opensuse/leap:15.0
 MAINTAINER Nathan Cutler <ncutler@suse.com>
 
-RUN zypper addrepo https://download.opensuse.org/repositories/filesystems:/ceph:/mimic/openSUSE_Leap_15.0/filesystems:ceph:mimic.repo \
+RUN zypper addrepo --refresh https://download.opensuse.org/repositories/filesystems:/ceph:/mimic/openSUSE_Leap_15.0/filesystems:ceph:mimic.repo \
```

Comment:
Just wondering why this is required in the base image, why not to include `zypper refresh` in rpm docker file?